pom.xml

```
@@ -119,13 +119,77 @@
                                 <version>8.29</version>
                             </dependency>
                         </dependencies>
-                        <configuration>
+                        <configuration>     
```

Please, restore the original formatting.
```suggestion
                        <configuration>
```

---

resolved